db/migrations/state/0014.sql

```diff
@@ -0,0 +1,6 @@
+-- +migrate Up
+CREATE INDEX IF NOT EXISTS idx_batch_global_exit_root ON state.batch (global_exit_root);
```

Comment:
I think the index should be (global_exit_root, batch_num) to improve the ORDER BY part of the query